addons/web/static/src/scss/notification.scss

```diff
@@ -15,7 +15,7 @@
 
             .o_notification_close {
                 height: 20px;
-                margin-left: 5px;
+                @extend .ml-3;
```

Comment:
Not a bad idea in general but should not be done with an `@extend` in this case:
- Only one property (and extend is a costly operation so not worth for an unique property IMO)
- Forced to important (as ml-3 is an important rule)
- This would generate:
`.ml-3, .o_notification_close { margin-left: ...; }` + `.o_notification_close { height: 20px; }` so two css rules for a small component as the notification close button, for 2 properties. In the end, your css in this case is bigger and the selectors more complex because of the extend.
- This component is a "backend" one but is used on the frontend side. This means that any bootstrap inside will adapt to the website theme css, which is not what you want in this case: you want to keep the same odoo UI. In general, this means we would have to force the correct margin size frontend side... but best having it hardcoded with a fixed PX value in the first place in this case.

Reply:
Ok, thanks for the clear explanation!